parser/testdata/workflows/applicationrequest-issue103.json

```diff
@@ -5,7 +5,7 @@
   "description": "Determine if applicant request is valid",
   "start": "CheckApplication",
   "specVersion": "0.7",
-  "auth": "./testdata/workflows/urifiles/auth.yaml",
+  "auth": "file://testdata/workflows/urifiles/auth.yaml",
```

Comment:
can't we keep without protocol?
maybe add just into one to make sure the unmarshal will work? wdyt?